### PR TITLE
fix earth science, SCIBUS, psych

### PR DESCRIPTION
**Pull request by kevinfwl**
Fixed inconsistencies in the aforementioned programs. The only problems now reside in chemistry but unfortunately, most (if not all) chemistry programs seem to have some sort of problem with it. I am taking a deeper look into their parsing but may need a bit more time. Let's merge this for now, while I continue investigating them.

### CODE REVIEW

File: ProgramParsing/Science/MajorParser2021_2022.py [hunk 1]
```diff
@@ -85,6 +86,18 @@ def _course_list(self, line, credit, oneOf = False):
             for c in courses:
                 if c not in d:
                     list.append(c)
+        
+        # handles: 1.25 BIOL units including BIOL 130, BIOL 130L, and 0.5 BIOL unit at 100- or 200-level (exclusive of BIOL 225 and BIOL 280)
+        if ("Psychology" in program_name and ("BIOL 225" in list or "BIOL 280" in list)):
```
Comment (haowei99) [inline ProgramParsing/Science/MajorParser2021_2022.py:91]: Is it safer to use` "psychology" in program_name.lower() `?

File: ProgramParsing/Science/MajorParser2021_2022.py [hunk 2]
```diff
@@ -237,6 +251,10 @@ def load_file(self, file, year):
 
             line = line.strip()
 
+            # hard coded for scibus because it double counts
+            if program == "Honours Science and Business" and "1.0 unit of 100- or 200-level BIOL" in line:
```
Comment (haowei99) [inline ProgramParsing/Science/MajorParser2021_2022.py:255]: hardcoding life is hard T-T

**Review by haowei99 — APPROVED** [review]
Let me know when you figure out alternative to SCI

**Comment by haowei99** [comment]
merging for now until chem can be resolved

File: ProgramParsing/Science/MajorParser2021_2022.py (hunk 1 above)
Comment (kevinfwl) [reply ProgramParsing/Science/MajorParser2021_2022.py:91]: Sorry I just saw this, I will fix this in the next PR

**Pull request merged**